lib/ueberauth/strategy/twitter.ex

```diff
@@ -105,13 +129,13 @@ defmodule Ueberauth.Strategy.Twitter do
       {:ok, %{status_code: 401, body: _, headers: _}} ->
         set_errors!(conn, [error("token", "unauthorized")])
       {:ok, %{status_code: status_code, body: body, headers: _}} when status_code in 200..399 ->
-        body = Poison.decode!(body)
+        body = @json_library.decode!(body)
```

Comment:
I see.  Upon seeing this - `module Ueberauth.Config is not available` - I thought we had a clashing namespace problem.  But once is added to the `ueberauth` library and update deps it will find it, correct?